packages/headless/src/controllers/result-list/headless-result-list.test.ts

```diff
@@ -14,6 +14,11 @@ describe('ResultList', () => {
     const results = new Array(10).fill(buildMockResult());
     engine.state.search.results = results;
     engine.state.search.response.totalCountFiltered = 1000;
+    jest.useFakeTimers('modern');
```

Comment:
With "modern", "Date.now()" is mocked.

Reply:
Ok, is it needed in this case though ?

Reply:
Yes, `fetchMoreResults` compares `Date.now()`.